src/datetime.rs

```diff
@@ -3,12 +3,16 @@
 
 //! ISO 8601 date and time with time zone.
 
-use std::{str, fmt, hash};
-use std::cmp::Ordering;
-use std::ops::{Add, Sub};
+use core::{str, fmt, hash};
+use core::cmp::Ordering;
+use core::ops::{Add, Sub};
+#[cfg(any(feature = "std", test))]
 use std::time::{SystemTime, UNIX_EPOCH};
 use oldtime::Duration as OldDuration;
 
+#[cfg(not(any(feature = "std", test)))]
+use alloc::string::{String, ToString};
```

Comment:
It seems like it might be worth sticking this (and what seems like the two functions that return strings) behind a separate `alloc` feature flag?

Reply:
I added an `alloc` feature now. I'm kind of sad how much is missing without it though. And it's mostly only because the timezone offset is stored as a String in the DelayedFormat.